front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/StorageServiceSupport.java

```diff
@@ -159,13 +199,16 @@ public void bulkImport(Collection<T> items) {
    * Update local cache with any recently modified items.
    */
   protected void refresh() {
-    long startTime = System.nanoTime();
-    allItemsCache.set(fetchAllItems(allItemsCache.get()));
-    long elapsed = System.nanoTime() - startTime;
+    long startTime = System.currentTimeMillis();
 
-    registry.timer("storageServiceSupport.cacheRefreshTime", "objectType", objectType.name()).record(elapsed, TimeUnit.NANOSECONDS);
+    cacheRefreshTimer.record(new Runnable() {
+      public void run() {
+          allItemsCache.set(fetchAllItems(allItemsCache.get()));
```

Comment:
any reason to create an inner runnable around this when we are already tracking the elapsed time (use .record(long instead of .record(Runnable) ?)